### DISCUSSION TITLE
Welcome to test Discussions!

### DISCUSSION THREAD

**Discussion by iammuho** [iammuho/test#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by diegomarangoni** [comment]
https://github.com/iammuho/test/blob/master/.github/mergeable.yaml#L3

**Reply by diegomarangoni** [reply]
test reply

**Comment by diegomarangoni** [comment]
test new conversation
